### DISCUSSION TITLE
Support for puppeteer?

### DISCUSSION THREAD

**Discussion by sawa-ko** [coollabsio/coolify#557] [Service Template Requests]
I am using the nodejs template and when trying to install puppeteer with yarn I am getting an error that the package could not be built succesfully.

![Screenshot_20220823-141406_Chrome.png](https://user-images.githubusercontent.com/56084970/186256917-26f44687-9aa1-44a3-879a-85c1024eecec.png)

**Comment by vasani-arpit** [comment]
Could be related to this https://github.com/puppeteer/puppeteer/issues/5611

Try with npm if possible.
